pwa-devdocs/src/pwa-buildpack/troubleshooting/index.md

```diff
@@ -95,6 +96,11 @@ Some users have suggested deleting the `devcert` folder to trigger certificate r
 ${User}\AppData\Local\devcert
   ```
 
+**Watcher runs out of memory**{:#watcher-out-of-memory}
+
+If you leave the `yarn watch` process running for more than 6 hours, it runs out of memory and shows an error.
```

Comment:
Was it 6 hours? I might just say "an extended amount of time". I think I've left it running overnight without issues.

So I might say "In some circumstances, leaving the watch process running for an extended amount of time can result in an out-of-memory error. The solution for this..."